zebrad/tests/common/get_block_template_rpcs/submit_block.rs

```diff
@@ -1,170 +1,88 @@
 //! Test submitblock RPC method.
 //!
-//! This test requires a cached chain state that is synchronized past the max checkpoint height,
-//! and will sync to the next block without updating the cached chain state.
-
-// TODO: Update this test and the doc to:
-//
-//         This test requires a cached chain state that is partially synchronized close to the
-//         network chain tip height, and will finish the sync and update the cached chain state.
-//
-//         After finishing the sync, it will get the first 20 blocks in the non-finalized state
-//         (past the MAX_BLOCK_REORG_HEIGHT) via getblock rpc calls, get the finalized tip height
-//         of the updated cached state, restart zebra without peers, and submit blocks above the
-//         finalized tip height.
-
-use std::path::PathBuf;
+//! This test requires a cached chain state that is partially synchronized close to the
+//! network chain tip height. It will finish the sync and update the cached chain state.
+//!
+//! After finishing the sync, it will get the first 20 blocks in the non-finalized state
```

Comment:
Nitpick: Let's be vague here, these docs don't match the current constant used in the code

```suggestion
//! After finishing the sync, it will get the first few blocks in the non-finalized state
```